lib/vmfloaty/ssh.rb

```diff
@@ -30,7 +30,12 @@ def self.ssh(verbose, host_os, token, url)
       end
 
       hostname = "#{response[host_os]["hostname"]}.#{response["domain"]}"
-      cmd = "#{ssh_path} #{user}@#{hostname}"
+      ssh_options= ["-oStrictHostKeyChecking=no", "-oUserKnownHostsFile=/dev/null"]
+      if ENV['VMFLOATY_SSH_KEY']
+        ssh_options << "-i #{ENV['VMFLOATY_SSH_KEY']}"
```

Comment:
Command line flag/config file/env. var. is the order of precedent I usually try to use in CLI apps. I haven't dug into `commander` -- does it map config file values to flags?